src/graphql/network.rs

```diff
@@ -309,16 +309,18 @@ struct HttpRawEvent {
     content_type: String,
     /// Cache Control
     cache_control: String,
-    /// Request Filenames
-    orig_filenames: Vec<String>,
-    /// Request MIME Types
-    orig_mime_types: Vec<String>,
-    /// Response Filenames
-    resp_filenames: Vec<String>,
-    /// Response MIME Types
-    resp_mime_types: Vec<String>,
-    /// POST Body
-    post_body: Vec<u8>,
+    /// Filenames
+    #[cfg_attr(feature = "cluster", graphql_client_type(from_name = "orig_filenames"))]
```

Comment:
Since peers connected through `PEER_VERSION_REQ: &str = ">=0.26.0-alpha.2,<0.26.0-alpha.3"` will share the same GraphQL schema, the attributes `#[cfg_attr(feature = "cluster", graphql_client_type(from_name = ...))]` for `filenames`, `mime_types`, and `body` are not necessary.

Reply:
The files `schema.graphql`, `http_raw_events.graphql`, and `network_raw_events.graphql`, which are used to forward requests from the Giganto to its peer Giganto, also need to be updated to reflect the changes in the HTTP-related fields.